query-service-impl/src/test/resources/postgres_service_request_handler.conf

```diff
@@ -1,7 +1,6 @@
 {
   name = raw-service-view-events-handler
   type = pinot
-  clientConfig = broker
```

Comment:
Note: In real config, we will have clientConfig with `connectionString` of postgres URL. we will need a way to accept password via config too.

Reply:
Need to define it as granular as we want